annotators/user_knowledge_memorizer/service_configs/user-knowledge-memorizer/environment.yml

```diff
@@ -7,3 +7,6 @@ TERMINUSDB_SERVER_PASSWORD: root
 TERMINUSDB_SERVER_TEAM: admin
 TERMINUSDB_SERVER_DB: user_knowledge_db
 CONFIG: config.json
+OPENAI_API_KEY:
```

Comment:
no. It is not done like it. See another examples using openai api. This key is stored in `.env_secret`